AnnotateUltrasound/AnnotateUltrasound.py

```diff
@@ -380,10 +382,20 @@ def onRemoveCurrentFrame(self):
         else:
             currentFrameIndex = self.logic.sequenceBrowserNode.GetSelectedItemNumber()
             self.logic.removeFrame(currentFrameIndex)
-            self.logic.updateLineMarkups()
-            ratio = self.logic.updateOverlayVolume()
-            if ratio is not None:
-                self._parameterNode.pleuraPercentage = ratio * 100
+
+            # Set programmatic update flag to prevent unsavedChanges from being set by updateLineMarkups
```

Comment:
L386-395 can by extracted into a helper fn since it also is used above in L1355-1366 below